airbyte-integrations/connectors/source-intercom/source_intercom/schemas/contacts.json

```diff
@@ -270,6 +270,40 @@
           "type": ["null", "boolean"]
         }
       }
+    },
+    "opted_out_subscription_types": {
```

Comment:
Let's make the same thing but for `"opted_in_subscription_types"` as well, once we add the opt_out we should add the opt_in, especially if it has the same structure

Reply:
Definitely, but can't do that until we update to Intercom API version 2.6 when that prop was added: https://developers.intercom.com/intercom-api-reference/v2.6/reference/changelog

Currently we're at 2.5, and I would not recommend trying to update version in this PR. In a near future PR I'll update to the latest version, 2.8, but there's one potential breaking change that Intercom notes in 2.7

Along with updating to 2.8 in that next PR, I'll attempt to add all the additional props + test coverage for them. Just wanted to get this PR through because 1. is my first time getting to know the codebase and 2. having this specific prop will really help my team :)